components/rhs_thread/rhs_thread.tsx

```diff
@@ -172,6 +172,19 @@ export default class RhsThread extends React.Component<Props, State> {
         }
     }
 
+    private userIsAtLastPost = () => {
+        const postReplies = (this.rhspostlistRef.current as HTMLElement)?.childNodes;
+        const rhsPostListRect = (this.rhspostlistRef.current as HTMLElement)?.getBoundingClientRect();
+        const offset = 24;
+
+        if (postReplies) {
+            const postRepliesArray = Array.from(postReplies);
```

Comment:
Is `Array.from` necessary here? `childNodes` isn't an array, but I think it still supports `.length` and indexing

Reply:
Sorry I left that change because I was using a filter before. I will update.